### ISSUE TITLE
Too many open files in tests

### ISSUE THREAD

**Issue by darkhelmet** [tcocca/active_pdftk#28]
Many things are not being cleaned up

**Comment by tcocca** [comment]
What dev environment are you on?  All specs are passing for us in Mac OSX 10.6 and Ubuntu 11.04.

**Comment by darkhelmet** [comment]
OSX 10.7 REE 1.8.7

**Comment by tcocca** [comment]
Hmm, I am also using ree 1.8.7 and we have tested in 1.9.2.  Unfortunately I haven't upgraded to 10.7 yet.  Have you pulled the commits I merged into master today?

**Comment by darkhelmet** [comment]
Yup. Trying to figure out if my open file  limit is just really low for some reason

**Comment by darkhelmet** [comment]
`ulimit -a` shows 256
